queue_job/README.rst

```diff
@@ -105,7 +105,7 @@ Configuration
   [options]
   (...)
   workers = 6
-  server_wide_modules = web,queue_job
+  server_wide_modules = web,queue_job,base_sparse_field
```

Comment:
To my knowledge, this is NOT needed when the field is imported directly instead of using the patched `odoo.fields.Serialized`.

Reply:
But I have shown you the capture with the error in our CI when we run the server without this:


![imagen](https://user-images.githubusercontent.com/7165771/184898605-0a0fd4cd-0760-4dc0-bf9c-6e817bde0c87.png)

Reply:
The code of `queue_job` is updated to latest one. Don't you think that not depending in the manifest on `base_sparse_field` is an error?

Reply:
And what's the problem with having such server wide module loaded?

Reply:
But one is attached to the other: if you don't add it to the server wide modules, then it's not loaded at all. And I insist: is there any problem adding to the server wide modules that I'm not aware of?

Reply:
>  if you don't add it to the server wide modules, then it's not loaded at all

This is not true. It was required WHEN we were using the patch.

The server wide module IS NOT needed. If you want to recommend it for certain cases, fine, but if you want to state in the docs that is required, to me is wrong. Unless proven otherwise. This is my point.

Reply:
Tested and indeed the server wide is not needed. @victoralmau please do a PR in 14.0 for removing such change in the documentation.